CORE-config_reader.py-19-__init__: current working directory :D:\Workspace\pyautomation
CORE-config_reader.py-20-__init__: using default config file: D:config\default.yml
CORE-config_reader.py-24-__init__: Exception occured [Errno 2] No such file or directory: 'D:config\\default.yml'
CORE-config_reader.py-25-__init__: Please put config file in [projectRoot]/config/default.yml or set environ variable AUTO_CONFIG
CORE-config_reader.py-19-__init__: current working directory :D:\Workspace\pyautomation
CORE-config_reader.py-20-__init__: using default config file: D:config\default.yml
CORE-config_reader.py-24-__init__: Exception occured [Errno 2] No such file or directory: 'D:config\\default.yml'
CORE-config_reader.py-25-__init__: Please put config file in [projectRoot]/config/default.yml or set environ variable AUTO_CONFIG
CORE-config_reader.py-19-__init__: current working directory :D:\Workspace\pyautomation
CORE-config_reader.py-20-__init__: using default config file: D:config\default.yml
CORE-config_reader.py-24-__init__: Exception occured [Errno 2] No such file or directory: 'D:config\\default.yml'
CORE-config_reader.py-25-__init__: Please put config file in [projectRoot]/config/default.yml or set environ variable AUTO_CONFIG
CORE-config_reader.py-19-__init__: current working directory :D:\Workspace\pyautomation\TestProject
CORE-config_reader.py-20-__init__: using default config file: D:\Workspace\pyautomation\config\default.yml
CORE-config_reader.py-24-__init__: Exception occured [Errno 2] No such file or directory: 'D:\\Workspace\\pyautomation\\config\\default.yml'
CORE-config_reader.py-25-__init__: Please put config file in [projectRoot]/config/default.yml or set environ variable AUTO_CONFIG
